vcd/resource_vcd_network.go

```diff
@@ -187,25 +208,28 @@ func resourceVcdNetworkCreate(d *schema.ResourceData, meta interface{}) error {
 			},
 			BackwardCompatibilityMode: true,
 		},
-		EdgeGateway: &types.Reference{
-			HREF: edgeGateway.EdgeGateway.HREF,
-		},
 		IsShared: d.Get("shared").(bool),
 	}
 
-	log.Printf("[INFO] NETWORK: %#v", newnetwork)
+	orgVDCNetwork.EdgeGateway = &types.Reference{
+		HREF: edgeGateway.EdgeGateway.HREF,
+	}
+
+	util.Logger.Printf("[INFO] NETWORK: %#v", orgVDCNetwork)
```

Comment:
Is this info really intended to be INFO level?

Reply:
I reused the logging calls from existing code, changing them to use our logger. Given the better logging that we have in go-vcloud-director, I could even remove these lines.